model-events/src/main/java/de/focus_shift/urlaubsverwaltung/extension/api/application/ApplicationDeletedEventDTO.java

```diff
@@ -0,0 +1,39 @@
+package de.focus_shift.urlaubsverwaltung.extension.api.application;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.UUID;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class ApplicationDeletedEventDTO {
+  @NonNull private UUID id;
+
+  @NonNull private Instant createdAt;
+
+  @NonNull private String tenantId;
+
+  @NonNull private ApplicationPersonDTO person;
+
+  @NonNull private ApplicationPersonDTO appliedBy;
+
+  private boolean twoStageApproval;
```

Comment:
Why is this interesting for consumers?

Reply:
i don't know all use case of potential consumers, why not providing the information?